samples/apps/smallbank/app/smallbank.cpp

```diff
@@ -21,6 +21,7 @@ namespace ccfapp
       "SmallBank_transact_savings";
     static constexpr auto SMALL_BANKING_AMALGAMATE = "SmallBank_amalgamate";
     static constexpr auto SMALL_BANKING_WRITE_CHECK = "SmallBank_write_check";
+    static constexpr auto SMALL_BANKING_SPIN = "SmallBank_spin";
```

Comment:
I think we should add this to our Batched app (which is already used for other kinds of stress test) rather than SmallBank.

Reply:
Good point, I'd forgotten the batched app. The reason I put it here is this going to run in perf, with worker threads, which is still something we need either way.